manual/manual.adoc

```diff
@@ -239,23 +242,26 @@ monsters with your bare hands. Ammo: None
 image:../sprites/csawa0.png[Ripsaw] |
 1 | Designed for cutting through wood, the ripsaw
 also works well as a melee weapon for cutting through flesh. Ammo: None
-| **Handgun** | 2 | Your starter weapon. A few shots will take down low-level
-monsters but it’s best to upgrade before taking on any tougher opponents.
+| **Handgun** +
+image:../sprites/pista0.png[Handgun] |
+2 | Your starter weapon. Its main purpose is to let you fight your way
+to a better weapon, and to hit shootable switches without wasting a second bullet.
 Ammo: Bullets
 | **Pump-action Shotgun** +
 image:../sprites/shota0.png[Pump-action Shotgun] |
-3 | The main “workhorse” weapon; fires seven pellets
-in a tight cluster and is effective over both short and long ranges against
-low- and medium-level monsters. Ammo: Shells
+3 | Shoots seven pellets in a fan pattern, letting you hit multiple
+targets or one big one. Ammo: Shells
 | **Double-barrelled Shotgun** +
 image:../sprites/sgn2a0.png[Double-barrelled Shotgun] |
-3 | Twice as powerful as the pump-action shotgun
-but less effective at long ranges; good at short range against crowds of
+3 | The simpler flash hiding system guarantees better
+projectile fragmentation for almost 50% more hits per shell
+across a wider spread. Good for short range against groups of
 enemies. Ammo: Shells
 | **Minigun** +
 image:../sprites/mguna0.png[Minigun] |
-4 | Conceptually like a faster version of the basic handgun, but
-eats ammo much more quickly. Ammo: Bullets
+4 | A much better use for your clips than the handgun.
```

Comment:
A bit confusing since clips aren't the only way to pick up bullets?

Reply:
..........do we really need to spell that out? I don't think someone who honestly can't infer that connection is going to get far enough through E1 to *see* a bullet box.

Reply:
That said, I will change it since this is actually a holdover from a lore thing I was trying to do to avoid calling full cartridges "bullets", which was never in the Spanish version of this sentence anyway.